app/javascript/components/components/Navigation.js

```diff
@@ -40,8 +40,13 @@ const Navigation = ({
         )}
         {logged_in && (
           <NavItem>
+<<<<<<< HEAD
```

Comment:
Looks like there still some stuff that needs to be cleared from the merge conflict. Lines 43 -49